src/main/resources/db/migration/V4.15.1.1__5028-dataset-explore.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE externaltool ADD COLUMN IF NOT EXISTS scope VARCHAR(255);
+UPDATE externaltool SET scope = 'FILE';
+ALTER TABLE externaltool ALTER COLUMN scope SET NOT NULL;
```

Comment:
should content type now be nullable (in the db and class definition) since it isn't used for scope dataset?

Reply:
As mentioned above, due to a bug in the pull request, dataset tools get the default content type. Yes, I think this field should be made nullable (and the bug fixed).

Reply:
So what is the status of this - is this being addressed? I.e., is the bug in ExternalToolServiceBean getting fixed in this PR, and is ContentType being made nullable?